src/constants/v1/ballotStrategies/index.ts

```diff
@@ -1,23 +1,29 @@
 import * as constants from '@ethersproject/constants'
 import { t } from '@lingui/macro'
+
 import { BallotStrategy } from 'models/ballot'
 
+import { SECONDS_IN_DAY } from 'constants/numbers'
+
 export function ballotStrategies(): BallotStrategy[] {
   return [
     {
       name: t`No strategy`,
       description: t`Any reconfiguration to an upcoming funding cycle will take effect once the current cycle ends. A project with no strategy may be vulnerable to being rug-pulled by its owner.`,
       address: constants.AddressZero,
+      durationSeconds: 0,
```

Comment:
nice! should we also add `durationSeconds` to the `BallotStrategy` type?

Reply:
it is on the type already as an optional

Reply:
🤦